packages/iTwinUI-react/src/core/Wizard/Wizard.test.tsx

```diff
@@ -234,15 +234,15 @@ describe('<Wizard />', () => {
 
     expect(document.querySelector('.iui-tooltip')).toBeNull();
     expect(screen.queryByText('Step one tooltip')).toBeNull();
-    userEvent.hover(screen.getByText('Step One'));
+    fireEvent.mouseEnter(screen.getByText('Step One'), { bubbles: true });
```

Comment:
We hover over span, but only parent `li` has the popover to trigger tooltip.
`userEvent` didnt have (or I didnt find) bubbling option, so used `fireEvent` here instead.